webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala

```diff
@@ -86,29 +86,13 @@ trait TechniqueWriter {
   ): IOResult[EditorTechnique]
 }
 
-/*
- * There is a configuration file option, `rudder.technique.compiler.app` that let you override
- * the choice for what app write techniques globally (techniqueCompilerApp).
- *
- * Then, you can locally override the choice for one technique with a local file `compilation.yaml`
- * in which you can force to fallback to `rudder` -> `rudder-only-unix` -> `webapp`.
- *
- * Finally, when `rudderc` is used, the webapp will look for its return code and generated files.
- * When the return code is XXXX, or if generated files doesn't pass a sanity check, then
- * the webapp is used to generate things. We also update the `compilation.yaml` to reflect the fact
- * that the last compilation used a fallback app and the error (if any)
- *
- * In that two last cases, then a big red persistent message is displayed in the technique editor
- * with the fact that a fallback was used.
- * In the last case, in addition, we need to display to the user the problem that they need to report
- * to rudder dev.
```

Comment:
why removing the whole comment ? If (parts of) is are outdate, you should rewrite and enhanced them with the real state of things, not just remove.

Reply:
I didn't see the `rudderc` part and I thought the documentation in the parent interface was already sufficient, but yes after double-checking, it's missing things about the FS like compilation-output.yaml

Reply:
:white_check_mark: https://github.com/Normation/rudder/pull/5905/commits/ed28eef83afc9fb4aef84d7d020a68ce36b4d63f